packages/ubuntu_provision/lib/src/services/theme_variant_service.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:ubuntu_logger/ubuntu_logger.dart';
+import 'package:yaru/yaru.dart';
+
+import '../../services.dart';
+
+/// @internal
```

Comment:
```suggestion
@internal
```
`meta` will have to be imported too.

Will this be used outside of this file? Otherwise it could just be made private.

Reply:
I just wanted to be consistent with all the other files that follow the exact same pattern. Is it worth changing this across the entire project?

Reply:
What does a dartdoc with @internal in it even mean? Surely that doesn't affect the analyzer? :thinking:

Reply:
I have no idea, it's probably unnecessary and doesn't do anything, since the `log` symbols need to be explicitly hidden from the barrel exports

Reply:
Let's make them private where possible and `@internal` at the other places then, I can do that in a separate PR. :)